src/App.tsx

```diff
@@ -16,36 +20,68 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortType, setSortType] = useState<SortType>(SortType.None);
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+
+  const resetGoods = (): void => {
```

Comment:
You don't have to define types everywhere, typescript understands what you return from the function